nltk/data.py

```diff
@@ -236,6 +243,7 @@ def normalize_resource_name(resource_name, allow_relative=True, relative_path=No
 ######################################################################
 
 class PathPointer(object):
+
```

Comment:
pep8 doesn't require a blank line before a docstring, and pep 0257 says: "There's no blank line either before or after the docstring. "